test/migration.test.js

```diff
@@ -338,6 +338,30 @@ describe('migrations', function() {
     });
   });
 
+  it('"mysql.columnName" is updated with correct name on create table', function (done) {
+    setupAltRowNameData();
+    db.autoupdate('ColRenameTest', function (err) {
+      db.discoverModelProperties('col_rename_test', function (err, props) {
+        assert.equal(props[0].columnName, 'first_name');
+        assert.equal(props[1].columnName, 'lastName');
+        assert.equal(props.length, 2);
+        done();
+      });
+    });
+  });
+
```

Comment:
Can remove redundancy and I think this test can be moved to the mysql.autoupdate.test.js file